src/utils/aepp-wallet-communication/rpc/aepp-rpc.js

```diff
@@ -151,6 +156,26 @@ export default Ae.compose({
       })
       return this.sendConnectRequest()
     },
+    /**
+     * Connect to node
+     * @function bridgeNode
+     * @instance
+     * @return {void}
+     */
+    async bridgeNode () {
+      if (!this.rpcClient || !this.rpcClient.isConnected()) throw new NoWalletConnectedError('You are not connected to Wallet')
+      const walletNode = await this.sendBridgeRequest()
+      if (!walletNode.url) {
+        throw new Error('Missing node url')
```

Comment:
Is it for a case when the request was rejected or if the wallet works not by protocol?

Reply:
If the wallet works not by the protocol